examples/custom_transform_and_model.ipynb

```diff
@@ -618,7 +618,7 @@
     }
```

Comment:
<p>I think we should describe what <code>ReversibleTransform</code> and <code>IrreversibleTransform</code>  is, and what is the difference between them. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Add the info about the difference</p>